getbraincloud/braincloud-csharp
# Develop (#75)

* quick rtt tests to test enableRTT and DisableRTT

* Major Update/ Sync - Remove Deprecated Get Singleton Mode, Clean up R… (#69)

* Major Update/ Sync - Remove Deprecated Get Singleton Mode, Clean up RTT services, Update lib websockets to source, so that we can have a modified version, that allows no delay to be enabled, which provides errors in ping and RTT bandwidth

* Ensure Use Unity Web Request. older WWW class is becoming obsolete....

* Add Async Match LIstener to RTT. a few missed to Lower() calls

* ensure all service and operation come in as TO LOWER

* UTIL updates for newest and latest unity, where TIMEZONE class is deprecated, and TIMEZONEINFO is used instead.

* commenting out RTT tests until properly working

* RTT now supports TCP and some UDP, UDP is always unreliable and non ordered at the moment,

* API calls, and tests. RS changes, and updated RTT tests. Still need to get RTT tests working, and add Lobby tests

* Removing the word plugin from the brainCloud Unity Settings

* Namespace wrapping

* Updating Unity Version

* Asset change

* Updating version number

* Meta data

* 4.0 API calls

* Here we go (#72)

* Here we go

* there we go

* try this udp connect to compile

* changes to proj setting for removed files

* Handling cleanup of old data

* More name changes

* Adding an import note to the readme

* Force adding dll

* adding braincloud namespace to JsonFx and LitJson csharp (#73)

* UniversalIdPassword resets

* sync to Relay APIs, updated new Lobby.CancelFindRequest .. fix up other issues in the unity section

* fix up the build

* Fixing merge errors

* Meta file

* updated fixes to c#

* more relay connections

* better comments

* isConnected via the relay service

* Removing empty update call

* minor refactors and updates

* Auth and Identity comms saving change (#74)

* Auth and Identity comms saving change

* adding missing check, and cleaning up

* comms fixes and refactors

* updating version for release
